components/LessonCard.tsx

```diff
@@ -27,50 +27,42 @@ type ReviewButtonProps = {
   className?: string
 }
 
-type ReviewCountProps = {
-  lessonId: number
-}
-
-const ReviewCount: React.FC<ReviewCountProps> = props => {
+const ReviewButton: React.FC<ReviewButtonProps> = props => {
   const { loading, data } = useQuery(GET_SUBMISSIONS, {
     variables: { lessonId: props.lessonId },
     fetchPolicy: 'network-only',
     nextFetchPolicy: 'cache-first'
   })
 
-  if (loading) {
-    return (
-      <div className="spinner-border spinner-border-sm mx-1" role="status" />
-    )
-  }
+  const style = `btn btn-sm btn-primary text-white float-end mb-2 me-2 ${
+    props.className || ''
+  }`
 
-  if (!data) {
-    return <span>0</span>
+  if (!props.isCompleted) {
+    return null
   }
-  const pendingSubmissionsCount = data.submissions.reduce(
-    (acc: number, val: any) => {
+
+  const pendingSubmissionsCount =
+    data &&
```

Comment:
we use lodash `get` to safety get properties from objects, we should the same here.